source/dofs/dof_handler_policy.cc

```diff
@@ -134,6 +134,146 @@ namespace internal
                            2*MultithreadInfo::n_threads(),
                            /* chunk_size = */ 32);
         }
+
+
+        typedef
+        std::vector<std::pair<unsigned int, unsigned int> > DoFIdentities;
+
+
+        /**
+         * Make sure that the given @p
```

Comment:
while here, would you mind making comments wider? I understand that's a copy-paste from previous code but i think it's worth fixing